### ISSUE TITLE
Serialization

### ISSUE THREAD

**Issue by PikaMug** [stefvanschie/IF#199]
Would you consider including an ability to serialize GUIs to/from the file system?

**Comment by stefvanschie** [comment]
I'm assuming you're talking about a way to turn a `Gui` instance to an XML file that can then be loaded again later?

**Comment by PikaMug** [comment]
Yes, XML or any other format. Ideally in a manner in which end users can change the slots of items to their liking.

**Comment by PikaMug** [comment]
If it helps a reference, [ObliviateInvs](https://github.com/hamza-cskn/obliviate-invs) and [MilkGUI](https://github.com/Browsit/MilkGUI) can do this.
